Cargo.toml

```diff
@@ -50,7 +50,7 @@ testing-fake-proofs = []
 m2-native = []
 
 [workspace]
-members = ["actors/*", "state", "runtime", "test_vm"]
+members = ["actors/*", "state", "runtime", "test_vm", "actors/evm/shared"]
```

Comment:
ugh, thats a minor annoyance.

Reply:
Yeah. We could also put the crate somewhere else, but it doesn't really matter (can easily move it).